sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
+
+## 1.0.0-beta.2 (Unreleased)
+
+
+## 1.0.0-beta.1 (2020-09-22)
```

Comment:
I think this should be beta.2. Should not it? All the packages should be incremented to 2 now

Reply:
Good catch. What about date? Remain 09-22?

Reply:
There's a typo (1.0.0-bata.1) in you comment @JianpingChen.

Reply:
If chat was never released on 9/22, just remove that line. And add a new unreleased line above it. Your changelog should look like:

```md
## 1.0.0-beta.3 (Unreleased)

## 1.0.0-beta.2 (2020-10-06)
- Initial release of Chat.
```

We will need to have a separate PR that goes into all the other libraries' changelogs to add the date for beta.2 and add the new unreleased line.